ALiAS_FAQs.md

```diff
@@ -0,0 +1,46 @@
+
+Q2. Why should I join ALiAS?
+Ans. ALiAS is by far, the largest open-source community in Amity University. We are
+open to developers not only inside the campus, but also from various places. We have got assistance
+from alumni and people from tech, near and far. We have got a presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
+
+Q3. How is the community of AliAS?
+And. We have the most active developers in the campus. You will find people from nearly all domains,
+Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people, designers, technical writers, theoretical CS enthusiasts,
+etc. And best of all, we have got a great reach for people and development.
+
+Q4. How should I join AliAS?
+Ans. Simple. Go and register yourself on http://asetalias.in/ . One of us will add you to the WhatsApp and Telegram groups.
+
+Q5. Having joined, where should I start?
+And. We request you to drop a message, stating your name, course duration, and your skills. Also
+let us know which field you are interested in, and we will guide you thereafter. Have one plan at a time.
+
+Q6. Best OS?
+Ans. Are you a designer? - Windows; Are you a developer? - Linux(Any)
+Windows has great graphics. It spoon feeds you everything. Many design softwares, games, drivers and
+support is easily available in windows. Installing anything is a cup of cake for anyone. But if you want
+to delve into softwares deeper, please forget your games and ease. Being open source community, we
+recommend installing Linux as dual boot, and get into ease with it. The only difference you will see in
+different distros is the GUI. Some software packages and libraries are pre-installed in some distros,
+while some requires you to install. Install Ubuntu or Linux Mint if you are a beginner. Ubuntu has got
+best support for you. (Pro-tip: Give more space to Home directory as all your files are installed here).
+
+Q7. The language I should start with?
+Ans. Start with C++. Next learn JAVA, then Python. This will fulfill most of your software needs and
+problems.
+
+P.S. - Please DON’T ASK TO ASK, JUST ASK. We all are learners here, and all of us are noobs in
+some field or the other. Don’t be shy. If you don’t know, just admit it, and ask and learn to be better
+you.
+
+Join us-
+email: contactasetalias@gmail.com
+GitHub: https://github.com/asetalias
+YouTube: https://www.youtube.com/channel/UCKkyqEMLFW3jz-q3nJIFL3g
+Facebook: https://www.facebook.com/asetalias/
+Mailing List: https://groups.google.com/forum/#!forum/amity-linux-group
```

Comment:
When did we get a mailing list?

Reply:
I tried to visit the link and I didn't have permission to see the content. 

Why do we have a mailing list that is not available publicly? and if it's private, should the link be added publicly to the FAQ!
cc: @vipulgupta2048 @thisisayush